src/api/index.ts

```diff
@@ -361,22 +361,21 @@ export function useApi(isOnline: boolean = true): BackendApi {
 									const user = await getItem("users", String(userId));
 									return {
 										data: {
-											session: {
-												id: user.id,
-												appToken: "",
-												did: user.did,
-												displayName: user.displayName,
-												privateData: user.privateData,
-												username: null
-											},
-											newUser: user,
+											id: user.id,
+											appToken: "",
+											did: user.did,
+											displayName: user.displayName,
+											privateData: user.privateData,
+											username: null,
+											webauthnUserHandle: user.webauthnUserHandle,
+											webauthnCredentials: user.webauthnCredentials
```

Comment:
It looks like `webauthnCredentials` is no longer needed since https://github.com/wwWallet/wallet-frontend/pull/264/commits/184afa200f053c82a342c23c598d66f8de7ae167. Should we drop it?